docs/specs/servers/blockchain/blockchain-bundler-api.md

```diff
@@ -0,0 +1,48 @@
+# Blockchain API Bundlers interaction
+
+This API provides a bundlers JSON-RPC interaction. 
+The supported `userOperation` interaction schema is according to v0.7 ERC-4337 specification.
+
+## Make a bundler JSON-RPC request
+
+Used to make a JSON-RPC call to the designated bundler.
+
+`POST /v1/bundler?projectId={projectId}&chainId={chainId}&bundler={bundlerName}`
+
+* `projectId` - Required. The project identifier.
+* `chainId` - CAIP-2 chain identifier.
+* `bundlerName` - (Optional) Bundler name to use.
```

Comment:
>`bundlerName` - (Optional)

is there a default bundler that will be used if the param is missing?

Reply:
The default bundler, for now, will be `Pimlico`. We can make this field required if that's a better approach.

Reply:
I changed this to be required since it doesn't makes sense to use default.